mso/resource_mso_schema_site_anp_epg.go

```diff
@@ -54,6 +54,20 @@ func resourceMSOSchemaSiteAnpEpg() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringLenBetween(1, 1000),
 			},
+			"private_link_label": &schema.Schema{
+				Type: schema.TypeSet,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
```

Comment:
Why ForceNew?

Reply:
This resource has no function for Update because all the parameters have ForceNew set as true. Hence, it has been set here.